### ISSUE TITLE
Black Template page after installing on Apple Silicon

### ISSUE THREAD

**Issue by akereachu** [fluxscape/fluxscape#37]
![Uploading Screen Shot 2024-06-11 at 4.37.23 PM.png…]()

**Comment by erictuvesson** [comment]
If you have something running on port 3000 when starting the editor it will look for the templates on that port, or it could be that you don't have any internet connection.

**Comment by alan-x-n** [comment]
I'm not able to recreate this issue on my end. Can you check if the what Eric said fixes things for you?

**Comment by akereachu** [comment]
Thank you  @erictuvesson and @alan-x-n, there's nothing running on port 3000, and my internet looks okay
I had thesame issue with the original repo, i thought fluxscape would have fixed the issue.

Please see same isssue here: https://github.com/noodlapp/noodl/issues/47

My Internet speed test below:
![Uploading Screen Shot 2024-06-12 at 11.42.56 AM.png…]()

**Comment by erictuvesson** [comment]
It's working fine for me, so I am unable to replicate the issue.

The editor does a request to `
https://docs.fluxscape.io/projecttemplates/index.json` to fetch the templates. This request could be blocked for some reason.

Could you share some logs like the console logs and network requests from the editor devtools? The editor devtools window can be opened with `Ctrl/Cmd + E`.

**Comment by akereachu** [comment]
<img width="1717" alt="Screen Shot 2024-06-12 at 6 14 57 PM" src="https://github.com/fluxscape/fluxscape/assets/62541622/f007317f-5190-40b2-9b0e-264afa7198e1">

**Comment by alan-x-n** [comment]
> <img alt="Screen Shot 2024-06-12 at 6 14 57 PM" width="1717" src="https://private-user-images.githubusercontent.com/62541622/339067942-f007317f-5190-40b2-9b0e-264afa7198e1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgyMTM1MDYsIm5iZiI6MTcxODIxMzIwNiwicGF0aCI6Ii82MjU0MTYyMi8zMzkwNjc5NDItZjAwNzMxN2YtNTE5MC00MGIyLTliMGUtMjY0YWZhNzE5OGUxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjEyVDE3MjY0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVkM2M1MjI2NGJlYmUyYjZlYTU1MTBkMDY3NTVkM2MxMzVhM2JlN2UxZTU1MGZmYjcyODM2OWZhZjU0NmEwMTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.zvhJ2GZeJfW6dU-g1sjkTULMn3_m1pWF4jmzADuqEMw">

Thanks for the feedback akereachu! it seems there's an old reference there, we will be patching it up soon and releasing the fix.

**Comment by akereachu** [comment]
Thanks for the feedback @alan-x-n, please let me know when to test again. can't wait to use the tool.

**Comment by erictuvesson** [comment]
@akereachu could you also share the network tab and select the request to `docs.fluxscape.io/projecttemplates/index.json`? I would like to see the request information like full endpoint etc.

What we see there is just a static name ("docs.noodl.net"), we should update this, but that is not affecting the logic.

The request is still made to `https://docs.fluxscape.io/projecttemplates/index.json`.

Here is the relative code:
https://github.com/fluxscape/fluxscape/blob/5d8b2d5bbab1547037e6921d3fd45e1d626f7ba3/packages/noodl-editor/src/editor/src/utils/forge/template/providers/noodl-docs-template-provider.ts#L7-L10

**Comment by erictuvesson** [comment]
If the request url looks like this in the editor devtools then it could be that the domain or IP is blocked, when you are not on a VPN. To figure out a solution I would need to see how this looks.

<img width="752" alt="image" src="https://github.com/fluxscape/fluxscape/assets/1263211/5203e5ec-43f0-4f1d-95c4-c5d411ecf27c">

_One thing that could be added in Fluxscape is error states when it can't load the file, but that wont solve the issue._

**Comment by akereachu** [comment]
I disconnected my VPN to test this is the screenshot
<img width="1685" alt="Screen Shot 2024-06-16 at 3 18 52 PM" src="https://github.com/fluxscape/fluxscape/assets/62541622/d0a6086a-186d-4955-8f20-6361f256a396">

**Comment by erictuvesson** [comment]
Are you able to open this link https://docs.fluxscape.io/projecttemplates/index.json in the browser?

**Comment by alan-x-n** [comment]
Yes, let us know if you're able to open that link. I had initially thought I saw the issue earlier in my comment, but it's unrelated to the issue you're posting.

**Comment by akereachu** [comment]
@alan-x-n @erictuvesson, I can access the link only when on VPN. so, I  think is a problem with my  ISP(Internet Service provider).

**Comment by alan-x-n** [comment]
> @alan-x-n @erictuvesson, I can access the link only when on VPN. so, I think is a problem with my ISP(Internet Service provider).

Very strange your ISP is blocking that, can I ask which ISP you have and country you're connecting from?

**Comment by akereachu** [comment]
@alan-x-n yeah very strange, ISP(MTN); Country(Cameroon).

**Comment by alan-x-n** [comment]
Out of curiosity, are you able to load this link? this is another an old docs by Noodl but hosted through GitHub pages, which im wondering if your ISP blocks for some reason. You may need to contact your ISP.

https://noodlapp.github.io/noodl-docs/

and then also try loading this.

https://fluxscape.github.io/fluxscape-docs/

and

https://docs.fluxscape.io/

**Comment by akereachu** [comment]
@alan-x-n i can't access the above Links without a VPN. i think i will keep using a VPN for now. until  my ISP fix the issue.
